torch/_inductor/lowering.py

```diff
@@ -753,11 +753,18 @@ def floor(x):
 
 
 @register_lowering(aten.round)
-def round(x):
-    if is_integer_type(x):
+def round(x, decimals=0):
```

Comment:
Actually, turns out we don't need this as it's already handled by a decomposition:
https://github.com/pytorch/pytorch/blob/0a7e2ce0e1abb8bbd0ee7084d112798211d6dbf9/torch/_inductor/decomposition.py#L183-L186

This decomposes `round.decimals` into `round.default` which in turn we lower.